### DISCUSSION TITLE
pdk cli should allow user to control the pdk log level

### DISCUSSION THREAD

**Discussion by JakeTRogers** [puppetlabs/pdk#1020] [Feature Request]
By default the PDK is set to log level INFO which leads to messages like this:
```
pdk (INFO): Running all available validators...
pdk (INFO): Using Ruby 2.4.4
pdk (INFO): Using Puppet 5.5.3
```
It would be ideal if there were a -l, --log-level CLI parameter which would accept the lowest level log entries the user wants displayed

Example use case: i would like to set the log level to WARN when it is called by my pre-commit hooks, which would decrease the number of lines I need to parse/look over.

**Comment by scotje** [comment]
Great idea @JakeTRogers ! I think this should be pretty simple to implement, would you have any interest in working on the code change if we provided some guidance? If not, we're more than happy to put it into our backlog and work on it when we can. :)

**Comment by JakeTRogers** [comment]
Sorry, didn't see the notification regarding your reply. Ill take look this weekend at the source and see if it looks like something I can manage. I would be glad to contribute if so.
